src/containers/Network/UpgradeStatus.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import axios from 'axios'
+import BarChartVersion from './BarChartVersion'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import Hexagons from './Hexagons'
+import { localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Log from '../shared/log'
+
+export const UpgradeStatus = () => {
+  const [vList, setVList] = useState<any>([{}])
+  const [validations, setValidations] = useState([])
+  const [unlCount, setUnlCount] = useState(0)
+  const [stableVersion, setStableVersion] = useState<string | null>(null)
+  const { t } = useTranslation()
+  const language = useLanguage()
+
+  const aggregateData = (v: any[]) => {
+    if (!v) {
+      return []
+    }
+    let total = 0
+    const tempData: any[] = []
+    v.reduce((res, val) => {
+      if (!res[val.server_version]) {
+        res[val.server_version] = {
+          server_version: val.server_version,
+          count: 0,
+        }
+        tempData.push(res[val.server_version])
+      }
+      res[val.server_version].count += 1
+      total += 1
+      return res
+    }, {})
+
+    return tempData
+      .map((item) => ({
+        label: item.server_version ? item.server_version : ' NA ',
+        value: (item.count * 100) / total,
+        count: item.count,
+      }))
+      .sort((a, b) => (a.label > b.label ? 1 : -1))
+  }
+
+  useEffect(() => {
+    fetchStableVersion()
+    fetchData()
+    const interval = setInterval(fetchData, 5000)
+    return () => {
+      clearInterval(interval)
+    }
+  }, [])
+
+  const fetchData = () => {
+    const url = '/api/v1/validators?verbose=true'
+
+    axios
+      .get(url)
+      .then((resp) => {
+        const newValidatorList: any = {}
+        resp.data.forEach((v: any) => {
+          newValidatorList[v.signing_key] = v
+        })
+
+        setVList(() => newValidatorList)
+        setUnlCount(resp.data.filter((d: any) => Boolean(d.unl)).length)
+      })
+      .catch((e) => Log.error(e))
+  }
+
+  const fetchStableVersion = () => {
+    const url = 'https://api.github.com/repos/XRPLF/rippled/releases'
+    axios.get(url).then((resp) => {
+      resp.data.every((release: any) => {
+        if (release.tag_name && !release.prerelease) {
+          setStableVersion(release.tag_name)
+          return false
+        }
+        return true
+      })
+    })
+  }
+
+  const updateValidators = (newValidations: any[]) => {
+    // @ts-ignore
```

Comment:
Why is there a `@ts-ignore` here? (Usually if we add thse, we include a comment on the same line explaining the purpose :) - it's also better to avoid wherever possible)